lib/main.coffee

```diff
@@ -137,9 +152,14 @@ module.exports =
           return messages.map (message) ->
             startPoint = [message.line - 1, message.column - 1]
             endPoint = [message.line - 1, message.column]
-            return {
+            ret = {
               type: message.type
-              text: message.message
               filePath,
               range: [startPoint, endPoint]
             }
+            if showSource
+              ret.html = '<span class="badge badge-flexible">' + (message.source or 'Unknown') + '</span> '
```

Comment:
I've never really found any documentation on the rules myself, so I'm not sure :confused:

The only place that I know of where any of this is documented is [here](https://github.com/squizlabs/PHP_CodeSniffer/wiki), but that doesn't have a section describing the rules themselves.